packages/client/TODO.md

```diff
@@ -93,13 +93,12 @@ Client manages the token lifecycle and storage
 
 ### Query publications
 
-- [Q] client.publication.fetch(request) or client.publications.fetchOne()
-- [PQ] client.publication.fetchAll(request)
-- [Q] client.publication.validateMetadata(request)
-- [PQ] client.publication.whoCollected({ publicationId })
-
-- [PQ] client.publication(s).forSale({ profileId })
-- [Q] client.publication.metadataStatus(request)
+- [x] [Q] client.publication.fetch(request) or client.publications.fetchOne()
+- [x] [PQ] client.publication.fetchAll(request)
+- [x] [Q] client.publication.validateMetadata(request)
+- [x] [PQ] client.publication.whoCollected({ publicationId })
+- [x] [PQ] client.publication(s).forSale({ profileId })
+- [x] [Q] client.publication.metadataStatus(request)
```

Comment:
notice the differences between what I suggested before

```
- [Q] client.publication.fetch(request)
- [PQ] client.publication.fetchAll(request)
- [Q] client.publication.validateMetadata(request)
- [PQ] client.publication.whoCollected({ publicationId })
- [PQ] client.publication.allForSale({ profileId })
- [Q] client.publication.metadataStatus(request)
```

all paginated queries could also use the pagination helper as it was merged already

there are two methods that I'd rename to make them more meaningful
- client.publication.**allWallets**WhoCollected (this one I missed before)
- client.publication.**all**ForSale